readme/INSTALL.md

```diff
@@ -1,5 +1,23 @@
-# Installation
-
+# Installation (CPU mode)
+
+install anaconda
```

Comment:
These instructions don't show up very well in the rendered file, since they aren't formatted in markdown